CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+- docs site - fix the fonts on `preset/presetname`. Issue #1128
+- docs site - add preview components on `preset/presetname`.
```

Comment:
Personally I feel like CHANGELOG should be just for the library, since it's something we hand write, & people can assume the docs are always getting better. I'm not sure if people are that curious about what specific upgrades we're making to docs if they're more casual users not looking at GitHub?

Reply:
> Are these borders supposed to look different though?

Not sure - already this broke once as useThemeUI stopped working and I had to replace it with useTheme from emotion.
I can check whats wrong with borders in a couple of days when I get some time - but it seems @hasparus doesn't like the implementation, so maybe easiest is to close this PR and let someone else take a stab?

Reply:
> Not sure - already this broke once as useThemeUI stopped working and I had to replace it with useTheme from emotion.

When did `useThemeUI` break?

Reply:
> it seems @hasparus doesn't like the implementation, so maybe easiest is to close this PR and let someone else take a stab?

It's not that I don't like it, I just think it's hard to get right. I think we could give it a stab starting from this point.

Reply:
> When did `useThemeUI` break?

In this PR - when the buttons stopped displaying their theme borders. I dont really know what happened tbh, just fixed by going upstream to emotion

Reply:
> It's not that I don't like it, I just think it's hard to get right. 

I don't mind either way - imo the best solution was to re-use the Components for all themes (its a pretty good piece of code to showcase any theme), and basically replace the more minimalist Lorem. 

For linking to other sites - this can be done additionally in credits.links. 
For creating mdx files for each theme - I really dont like it.